Forge.TreeWalker/src/ITreeWalkerCallbacks.cs

```diff
@@ -14,7 +14,9 @@ namespace Microsoft.Forge.TreeWalker
     using System.Threading.Tasks;
 
     /// <summary>
+    /// [DEPRECATED]
     /// The ITreeWalkerCallbacks interface defines the callback Tasks that are awaited while walking the tree.
+    /// Prefers future callers to use the new ITreeWalkerCallbacksV2, and keep this ITreeWalkerCallbacks for backward compatibility.
```

Comment:
>, and keep this ITreeWalkerCallbacks for backward compatibility [](start = 68, length = 63)

Remove this comment.

Reply:
removed

---
In reply to: [506628449](https://github.com/microsoft/Forge/pull/32#discussion_r506628449) [](ancestors = 506628449)